scripts/handlebars/templates/templates.hbs

```diff
@@ -1,65 +1,7 @@
+Templates are deprecated in favor of [Formats](formats.md) and will be removed in the future. If you want to create a custom template, create a [Format that uses templating](formats.md#using-a-template-templating-engine-to-create-a-format).
 
-Templates and Formats serve the same purpose: use your style dictionary as data to build a file. You use templates in your config
-file under platforms > [platform] > files > [file]
+>*__Why are Templates being deprecated in favor of Formats?__*
 
-```json
-{
-  "source": ["properties/**/*.json"],
-  "platforms": {
-    "android": {
-      "transformGroup": "android",
-      "files": [
-        {
-          "template": "android/colors",
-          "destination": "colors.xml"
-        }
-      ]
-    }
-  }
-}
-```
-
->*__How are Templates different than Formats?__*
-
->Mainly syntactic sugar; anything you can do in a Template you can do in a Format. Use whichever is easier for you to write. We find
-that Templates are good if you have a lot of boilerplate code around where the style dictionary will go (like writing ObjectiveC files).
-Formats are better if there is little to no boilerplate code like a flat SCSS variables file.
-
-### Using a different templating language
-
-If you are partial to Handlebars or some other templating language, anything will work as long as there is a node module for it.
-All you need to do is register a custom format which calls your template and returns a string. Here is a quick example for Handlebars.
-
-```js
-const StyleDictionary = require('style-dictionary').extend('config.json');
-const Handlebars = require('handlebars');
-
-const template = Handlebars.compile( fs.readFileSync('templates/MyTemplate.hbs') );
-
-styleDictionary.registerFormat({
-  name: 'myTemplate',
-  formatter: function(dictionary, platform) {
-    return template({
-      properties: dictionary.properties,
-      options: platform
-    });
-  }
-});
-
-styleDictionary.buildAllPlatforms();
-```
-
-----
-
-## Pre-defined Templates
-
-[lib/common/templates.js](https://github.com/amzn/style-dictionary/blob/master/lib/common/templates.js)
-
-{{#namespace name="Templates"}}
-{{>members~}}
-{{/namespace}}
+>It is a simplification for users, no power is being removed. Templates were syntactic sugar; anything you could do in a Template you can do in a Format. Since they both did the same thing people became confused on which method they were supposed to use for output. Even worse, we actually required you to know if the output you wanted was created as a format or as a template, even for those formats and template included in Style Dictionary by default. This was a bad plan and caused problems. Additionally, Style Dictionary aims to provide power and flexibility without forcing you to use any particular system and only Lodash Templates were supported under the old system. Using formats, you can use any Templating engine you would like.
```

Comment:
We can just delete this handlebars file b/c it is not doing anything anymore. The markdown file it creates is checked  in and we don't need to regenerate it when we rebuild the docs.